computations/x1_16_chabauty.m

```diff
@@ -43,63 +55,72 @@ function EllipticChabauty(C, g, hk, HktoA)
   print "gamma", gamma;
   gamma_g := <gamma,g>;
 
+  // the promised elliptic curve above
   E := HyperellipticCurve(gamma*g);
+
+  // However, we need it as an EllipticCurve object, so we do the following
   EtoP1:=map<E->P1|[E.1,E.3]>;
   iselliptic, E1raw, EtoE1raw, E1rawtoE := IsEllipticCurve(E);
+
+  // If it is not elliptic, we need to find a point at infinity
   if not iselliptic then
     print "finding points";
     time Epoints := Points(E : Bound:=100);
-
+    
+    // If we can't find a point, we return
```

Comment:
```suggestion
    // If we can't find a point, we return
    // technically we could try to proceed and show that E has no points,
    // but since E is covered by an everywhere locally solvable curve
    // this will not be easy.
    
```

Reply:
done